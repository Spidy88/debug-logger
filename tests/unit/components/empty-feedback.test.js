import React from 'react';
import renderer from 'react-test-renderer';
import EmptyFeedback from '../../../client/components/empty-feedback';

describe('<EmptyFeedback>', () => {
    it('should render the feedback component', () => {
        const icon = 'barber';
        const message = 'Test message';
        const subMessage = 'A message below the main message';

        const tree = renderer.create(
            <EmptyFeedback icon={icon} message={message} subMessage={subMessage} />
        );

        expect(tree.toJSON()).toMatchSnapshot();
    });
});