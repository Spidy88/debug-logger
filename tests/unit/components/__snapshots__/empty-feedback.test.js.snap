// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyFeedback> should render the feedback component 1`] = `
<div
  className="ðŸ“¦pb_1rem ðŸ“¦pt_1rem ðŸ“¦dspl_flex ðŸ“¦flx-drct_column ðŸ“¦algn-itms_center ðŸ“¦just-cnt_center ðŸ“¦box-szg_border-box css-nil"
>
  <svg
    data-file-name="SvgUndraw_barber_3uel"
    height="20rem"
    width="20rem"
  />
  <span
    className="ðŸ“¦color_425A70 ðŸ“¦fnt-fam_b77syt ðŸ“¦fnt-sze_2rem ðŸ“¦f-wght_500 ðŸ“¦ln-ht_2-5rem ðŸ“¦ltr-spc_-0-05px ðŸ“¦mb_12px ðŸ“¦box-szg_border-box"
  >
    Test message
  </span>
  <span
    className="ðŸ“¦color_425A70 ðŸ“¦fnt-fam_b77syt ðŸ“¦fnt-sze_1-5rem ðŸ“¦f-wght_300 ðŸ“¦ln-ht_1-75rem ðŸ“¦ltr-spc_-0-05px ðŸ“¦box-szg_border-box"
  >
    A message below the main message
  </span>
</div>
`;
